config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/api/v1/genres", to: "api/v1/genres#index"
+  get "/api/v1/songs", to: "api/v1/songs#index"
```

Comment:
currently FE is sending requests as a `POST` to `playlists`. User stories 3 and 4 will refactor to provide the `POST`  `playlists` route, that will call the `POST` `songs` route.